portfolio_style.css

```diff
@@ -0,0 +1,155 @@
+@import url('https://fonts.googleapis.com/css?family=Charm|Fredericka+the+Great');
+
+html {
+	font-family: "Fredericka the Great";
+	color: #5f6161;
+	font-size: 10pt;
```

Comment:
better to use 10px here instead of pt

and then you can go on to use rem for rest of the sizing on the page

Reply:
Ok, you are right. Changed it